src/ckms.rs

```diff
@@ -64,13 +67,25 @@ impl<T> AddAssign for CKMS<T>
             (Some(x), None) => Some(x),
             (Some(x), Some(y)) => Some(x.add(y)),
         };
+        self.cma = match (self.cma, rhs.cma) {
```

Comment:
Sorry, been awhile.

What is the significance of setting and referencing self.cma here vs say the value of rhs?